### PR TITLE
cli: remove --print-config

### PR DESCRIPTION
**Pull request by adamraine**
This is our only usage of `generateConfig` #14547

### CODE REVIEW

File: cli/test/cli/index-test.js [hunk 1]
```diff
@@ -80,81 +80,4 @@ describe('CLI Tests', function() {
...
-      assert.strictEqual(config.settings.output[0], 'json');
-      assert.strictEqual(config.settings.auditMode, true);
-      assert.strictEqual(config.audits.length, 1);
-
-      expect(config).toMatchSnapshot();
-    });
-  });
-
-  describe('preset', () => {
-    it('desktop should set appropriate config', () => {
```
Comment (paulirish) [inline cli/test/cli/index-test.js:123]: would be good to still test this. 
maybe a mock for the core/ API and check the config?

Comment (adamraine) [reply cli/test/cli/index-test.js:123]: It's tested in bin-test.js

Comment (paulirish) [reply cli/test/cli/index-test.js:123]: ah.. the experimental preset. true

**Comment by connorjclark** [comment]
🫡

**Pull request merged**